internal/parse/parse.go

```diff
@@ -62,7 +62,7 @@ func EnhanceAST(name string, content string, module *ast.Module) (map[string]any
 	enhancedAst["regal"] = map[string]any{
 		"file": map[string]any{
 			"name":  name,
-			"lines": strings.Split(content, "\n"),
+			"lines": strings.Split(strings.ReplaceAll(content, "\r\n", "\n"), "\n"),
```

Comment:
I wonder if it'd be faster to use a Regex? `\r?\n` (rather than 2 replacements)

Also, a comment to explain that this specifically to handle windows line endings would be welcome since that might not be immediately clear.

Reply:
Thanks! I'll add a comment in a follow-up PR. About the regex, I guess we'd need to measure 😅